### PR TITLE
Flesh out Hid Injector Sample

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

File: HidInjector/app/HidInjectorTest.c [hunk 1]
```diff
@@ -222,11 +207,11 @@ SendTestInput(
 		InjectKeyDown(VK_RETURN);
 		InjectKeyUp(VK_RETURN);
 	}
+    */
```
Comment (alecont) [inline HidInjector/app/HidInjectorTest.c]: why all these comments? are they useful? or should we just clean them up?

File: HidInjector/driver/HidInjectorKd.vcxproj.Filters [hunk 2]
```diff
@@ -38,13 +38,13 @@
     </ClInclude>
   </ItemGroup>
   <ItemGroup>
-    <Inf Include="HidInjectorKd.inx">
-      <Filter>Driver Files</Filter>
-    </Inf>
-  </ItemGroup>
-  <ItemGroup>
     <ResourceCompile Include="HidInjectorKd.rc">
```
Comment (alecont) [inline HidInjector/driver/HidInjectorKd.vcxproj.Filters:41]: This is still "HidInjectorKd" with "Kd", is that ok?

**Comment by alecont** [comment]
couple of minor comments.

File: HidInjector/app/HidInjectorTest.c (hunk 1 above)
Comment (ooeygui) [inline HidInjector/app/HidInjectorTest.c]: I'll add a note. These are how you'd send these messages, but doing them all together they really make no sense.

File: HidInjector/driver/HidInjectorKd.vcxproj.Filters (hunk 2 above)
Comment (ooeygui) [inline HidInjector/driver/HidInjectorKd.vcxproj.Filters:41]: I renamed it. This is the removed one.

**Comment by alecont** [comment]
great thx!

**Pull request merged**